### ISSUE TITLE
Getting error on MacOS Sonoma 14.4.1

### ISSUE THREAD

**Issue by alarmatwork** [dolphin-emu/WiimotePair#5]
Pairing Error: An error occurred while attempting to pair: "(os/kern) default set".

**Comment by ajbarrett715** [comment]
I've got that same error, not sure how to fix it. Let me know if there's any updates

**Comment by Yoyopizzacat** [comment]
I got the same error on MacOS Sonoma and then I updated to Sequoia 15 Beta in hopes that it would make it work but no luck. Maybe I will try booting Ventura or Monterey from external hard drive but idk.

**Comment by leafysnowylemur** [comment]
I was able to get rid of the error and was able to successfully pair Wiimotes to my M1 MacBook Air running macOS 14.3.1. I'm not sure how to reliably pair Wiimotes at this point, but I'm working on it.  Running '''sudo pkill bluetoothd''' in the terminal to reset the bluetooth module helped, I think. It was able to pair through WiimotePair after that. At one point I was able to pair 3 Wiimotes in succession within the span of 20 seconds simply by having the app open and pressing the sync button. I haven’t been able to recreate this yet, but I'm working on it.

**Comment by Yoyopizzacat** [comment]
I ended up ordering a Bluetooth adaptor off aliexpress. Hope it works but I will try reseting the Bluetooth module.

**Comment by leafysnowylemur** [comment]
I made a reddit post on the Dolphin Emulator subreddit detailing my process of connecting Wiimotes, which is actually a relatively reliable method of being able to connect to them (If you consider being able to connect to them if you spend 10 minutes trying "reliable") 

https://www.reddit.com/r/DolphinEmulator/comments/1dwkq3m/i_was_able_to_pair_wiimotes_to_macos_1431_with/

If you're up for some messing about, you can try it and see if it works. I sill can’t get it very consistent though. I'm tempted go through the WiimotePair app in Xcode on my own to see what's going on and why doing what I do with blueutils and forcing a connection does what it does.

**Comment by Yoyopizzacat** [comment]
Thank you so much! I will try later.

**Comment by Klikini** [comment]
@leafysnowylemur I tried your method on my M3 MacBook Pro running Sonoma 14.5 about 5 times and it wasn't successful. I wonder is something changed between M1 and M3, or macOS 14.3.1 and 14.5?

**Comment by MoeSiz** [comment]
I'm on OS 14.6 and resetting the bluetooth module let me pair a wii remote, but it will not connect properly.

**Comment by Lewiscowles1986** [comment]
So for me it pairs, but also when I look at the device, it says the driver is magic trackpad

**Comment by Lewiscowles1986** [comment]
At this point I'm wondering if it's worth pairing on a mac anyway, or if I should just use a raspberry pi or something. Are they powerful enough for Dolphin?

**Comment by leafysnowylemur** [comment]
> So for me it pairs, but also when I look at the device, it says the driver is magic trackpad

Yes. Wiimotes, for whatever reason, show up as a Magic Trackpad.

**Comment by leafysnowylemur** [comment]
> At this point I'm wondering if it's worth pairing on a mac anyway, or if I should just use a raspberry pi or something. Are they powerful enough for Dolphin?

I don’t think so. Maybe a Pi5, but I doubt it.

**Comment by Yoyopizzacat** [comment]
Just buy this or something similar. Worked very well. 
TP Link UB400
https://www.amazon.com.au/dp/B07NQ5YGDW

**Comment by Klikini** [comment]
I second the UB400; using it with Dolphin’s Bluetooth passthrough feature allowed me to pair a Wiimote with my MacBook Pro M3.

**Comment by Lewiscowles1986** [comment]
okay, I'm not in Australia so the Amazon link won't work for me; but also $20... wow.

Does anyone know the chipset that U400 uses? And the minimum bluetooth needed to support a wii controller?

**Comment by Yoyopizzacat** [comment]
I bought it for 12 AUD which is about 8 USD so not unreasonable at all.

**Comment by alfonsico** [comment]
> I made a reddit post on the Dolphin Emulator subreddit detailing my process of connecting Wiimotes, which is actually a relatively reliable method of being able to connect to them (If you consider being able to connect to them if you spend 10 minutes trying "reliable")
> 
> https://www.reddit.com/r/DolphinEmulator/comments/1dwkq3m/i_was_able_to_pair_wiimotes_to_macos_1431_with/
> 
> If you're up for some messing about, you can try it and see if it works. I sill can’t get it very consistent though. I'm tempted go through the WiimotePair app in Xcode on my own to see what's going on and why doing what I do with blueutils and forcing a connection does what it does.

I got it work with Mac mini m2 15.7 but I update my Mac to Mac mini m2 pro. I have tried with 26 and look impossible, I downgrade to 15.7 in Mac mini m2 pro and I can't replicate the sync of Wiimote and I'm sure I got it "without" problems some months ago

I buy a BT 4.0 tp link to passthrough and works but I guess I prefer the other method to mix controls emulate and real
